.gitignore

```diff
@@ -198,3 +198,6 @@ artifacts/
 python_subproc_tests.py
 *.pdf
 *.ps
+
+# Exclude vscode dev container docker files
+.docker/*
```

Comment:
@yanii why files are being generated in this dir? I'm curious why my workstations / compute nodes have not created this directory.